src/frontend/magnify/src/views/rooms/settings/RoomSettingsView.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { rest } from 'msw';
+import React from 'react';
+import { createMemoryRouter } from 'react-router-dom';
+import { MagnifyTestingProvider } from '../../../components';
+import { roomConfigMessages } from '../../../components/rooms/RoomConfig/RoomConfig';
+import { defaultRoom } from '../../../mocks/handlers/rooms/roomsHandlers';
+import { server } from '../../../mocks/server';
+import { buildApiUrl } from '../../../services/http/http.service';
+import { RoomSettingsView } from './index';
+
```

Comment:
you can rename this file to `index.spec.tsx` ( the same principles applies to all other similar case in this PR )